src/components/ItemSelector/SelectedItems.js

```diff
@@ -17,6 +18,14 @@ const Subtitle = () => (
     </div>
 )
 
+const InfoBox = ({ message }) => (
+    <div className="info-container">
+        <InfoIcon style={{ fontSize: 16, color: '#6E7A8A' }} />
```

Comment:
Is the color something we can bring in from ui-core?

Reply:
I think so, there is a colors file with some definitions, that can probably be removed altogether and use ui-core colors instead.